backends/cxxrtl/runtime/cxxrtl/cxxrtl.h

```diff
@@ -849,9 +763,9 @@ std::ostream &operator<<(std::ostream &os, const value_formatted<Bits> &vf)
 			if (val.is_zero())
 				buf += '0';
 			while (!val.is_zero()) {
-				value<Bits> quotient;
-				val.divideWithRemainder(value<Bits>{10u}, quotient);
-				buf += '0' + val.template trunc<(Bits > 4 ? 4 : Bits)>().val().template get<uint8_t>();
+				value<Bits> quotient, remainder;
+				std::tie(quotient, remainder) = val.template divmod_uu<Bits>(value<Bits>{10u});
```

Comment:
The dependency here is a bit in the wrong direction: `divmod_uu` (a Yosys cell implementation) should depend on something in `value<>` (the generic library), not the other way around.

I would also like to have a `divmod` method on `value<>` to use elsewhere.

Reply:
Done. Location of `value<>::divmod` is due to reference to `value<32>` and `value<1>`, thus necessitating the definition to come after `expr_base`. Open to suggestions on a better location.